pkg/services/org/orgtest/fake.go

```diff
@@ -85,7 +91,9 @@ func (f *FakeOrgService) GetOrgUsers(ctx context.Context, query *org.GetOrgUsers
 }
 
 func (f *FakeOrgService) RemoveOrgUser(ctx context.Context, cmd *org.RemoveOrgUserCommand) error {
-	return f.ExpectedError
+	testData := f.ExpectedOrgListResponse[0]
```

Comment:
this is a nitpick, it should not block merging if you don't agree (or even don't feel like changing it right now):

I am concerned about this particular Fake - this behavior is inconsistent compared to the other fakes, which can cause confusion for future devs. I try not to nitpick tests in general, but in my experience inconsistencies such as this one can make refactoring (or writing) tests disproportionately harder or slower.

If a test fake needs logic beyond returning an expected result and/or error, we're better served by adding an RemoveOrgUserFn to the Fake struct that the test could optionally define.

I don't think I'm describing this well, so here's an example:
https://github.com/grafana/grafana/blob/main/pkg/services/user/usertest/fake.go#L64

Reply:
I copied this from the store mocks, this as all our fakes should be refactored